projects.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Personal Homepage</title>
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <header class="c-site-header">
+      <h1 class="c-site-header__title">Fathi Kahin</h1>
+      <p class="c-site-header__subtitle">(Upcoming Software Developer)</p>
+    </header>
+
+    <nav aria-label="Main Site Links" class="c-site-nav">
+      <a href="index.html" class="c-site-nav__link">Home</a>
+      <a href="/about.html" class="c-site-nav__link">About</a>
+      <a href="/projects.html" class="c-site-nav__link">My projects</a>
+      <a href="/contact.html" class="c-site-nav__link">Contact</a>
+    </nav>
+
+    <main>
+   <section>
+  <h1 class="c-section__title">List of my Projects</h1>
```

Comment:
It is considered a good practice to have only one `<h1>` per page. The `<h1>` tag represents the `main heading` of the page and holds significant importance for search engine optimisation and accessibility.

Instead, consider using `<h2>, <h3>`, and other heading tags to create a **hierarchical** structure for your headings.